azext_iot/_help.py

```diff
@@ -197,6 +197,13 @@
     short-summary: Export all device identities from an IoT Hub to an Azure Storage blob container.
     long-summary: For more information, see
                   https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities
+    examples:
+    - name: Export all device identities to blob container using SAS uri.
```

Comment:
"Export all device identities to a configured blob container using an inline SAS uri."